22146.7 <c-16#131> Created logger
22146.8 <c-16#131> appl_initialize was called
22146.8 <c-16#131:4> Logger was updated
22146.8 <c-16#131:4> Started intializing cube lib
22146.8 <c-16#131:4> Finished intializing cube lib with worker, starting to work
22147.6 <c-16#131:4> requestCubes => [43] Buffer size: 0
22147.6 <c-16#131:4> Sent requestCubes signal to root
22147.6 <c-16#131:4> Received 1 cubes from root
22147.6 <c-16#131:4> The main loop continues
22147.6 <c-16#131:4> Started solving a cube
22147.6 <c-16#131:4> Cube failed
22147.6 <c-16#131:4> Added failed cube
22147.6 <c-16#131:4> returnFailedCubes => [43] Buffer size: 3
22147.6 <c-16#131:4> Sent 1 failed cubes to root
22147.6 <c-16#131:4> Received receivedFailedCubes signal from root
22147.6 <c-16#131:4> requestCubes => [43] Buffer size: 0
22147.6 <c-16#131:4> Sent requestCubes signal to root
22147.6 <c-16#131:4> Received 1 cubes from root
22147.6 <c-16#131:4> The main loop continues
22147.6 <c-16#131:4> Started solving a cube
22960 <c-16#131:4> appl_interrupt was called
22960 <c-16#131:4> Interrupting worker
22960 <c-16#131:4> appl_interrupt was called
22960 <c-16#131:4> appl_withdraw was called
22960 <c-16#131:4> Started withdraw thread
22960.1 <c-16#131:4> STOP (0.15s since last cb)
22960.1 <c-16#131:4> Solving interrupted
22960.1 <c-16#131:4> The main loop continues
22960.1 <c-16#131:4> Leaving the main loop due to an interruption
22960.1 <c-16#131:4> Finished withdrawing lib
22960.1 <c-16#131:4> Enter destructor of CubeWorker
22960.1 <c-16#131:4> Time waiting for messages: 0.002
22960.6 <c-16#131:4> Enter destructor of CubeWorkerInterface
22960.6 <c-16#131:4> Enter destructor of CubeRoot
22960.6 <c-16#131:4> Finished withdraw thread
23021.2 <c-16#131:4> Enter destructor
23021.2 <c-16#131:4> Time working: 813.258
23021.2 <c-16#131:4> Time suspended: 0.000
23021.2 <c-16#131:4> Joined cleanup thread
23021.2 <c-16#131:4> Exit destructor
23021.2 <c-16#131:4> Destructing logger
